ProcessLib/TES/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+if(OGS_EIGEN_DYNAMIC_SHAPE_MATRICES)
+	configure_file(
```

Comment:
Not sure about the config files, but I also formatted them to be w/o tabs in the other PR.